lib/tether/tether.go

```diff
@@ -290,6 +290,9 @@ func (t *tether) Register(name string, extension Extension) {
 func (t *tether) handleSessionExit(session *SessionConfig) {
 	defer trace.End(trace.Begin("handling exit of session " + session.ID))
 
+	// notify guest shutdown waiter this process has gone away
+	close(session.exit)
```

Comment:
can we move this after the IO is shut down - if were to hang anything else off the channel, whether all output has been written could be important.